src/ol/control/attributioncontrol.js

```diff
@@ -44,7 +43,7 @@ ol.control.Attribution = function(opt_options) {
   this.logoLi_ = document.createElement('LI');
 
   this.ulElement_.appendChild(this.logoLi_);
-  goog.style.setElementShown(this.logoLi_, false);
+  this.logoLi_.style.display = 'none'; // Hide the logo
```

Comment:
This `// Hide the logo` comment and similar comments throughout this pull request could be removed, if you get a chance.